openbas-api/src/main/java/io/openbas/migration/V2_71__Modify_inject.java

```diff
@@ -41,7 +41,7 @@ constraint injects_asset_groups_pkey primary key (inject_id, asset_group_id)
         """);
     // Add asset to inject expectation
     select.execute("""
-        ALTER TABLE injects_expectations ADD COLUMN inject_expectation_group bool;
+        ALTER TABLE injects_expectations ADD COLUMN inject_expectation_group bool default false;
```

Comment:
Why not add another migration script with this update? That should generate a checksum error, right?

Reply:
Because we don't have a release with this migration in it.
So we can do the changes direclty here.

Reply:
thanks!